UI/auth-youtube.hpp

```diff
@@ -11,15 +11,22 @@
 #include "window-dock-browser.hpp"
 #include "lineedit-autoresize.hpp"
 #include <QHBoxLayout>
+#include <QTimerEvent>
+
 class YoutubeChatDock : public BrowserDock {
 	Q_OBJECT
 
 private:
 	std::string apiChatId;
+	int awaitLoginTimer;
+	bool isLoggedIn;
 	LineEditAutoResize *lineEdit;
 	QPushButton *sendButton;
 	QHBoxLayout *chatLayout;
 
+protected:
```

Comment:
Do not use `protected` - its use is discouraged in modern C++ and by the ISO C++ guidelines.

Use `private` instead.

Reply:
This is overriding the default QObject implementation, which is defined as a `protected` method. See https://doc.qt.io/qt-6/qobject.html#timerEvent